### PR TITLE
refactor: namespace repository, handle encryption in service

### PR DESCRIPTION
**Pull request by rahmatrhd**

### CODE REVIEW

File: pkg/namespace/service.go [hunk 1]
```diff
@@ -157,3 +137,34 @@ func (s Service) DeleteNamespace(id uint64) error {
 func (s Service) Migrate() error {
 	return s.repository.Migrate()
 }
+
+func (s Service) toEncryptedNamespace(ns *domain.Namespace) (*domain.EncryptedNamespace, error) {
```
Comment (ravisuhag) [inline pkg/namespace/service.go:141]: @rahmatrhd method name could be just `encrypt` and `decrypt`.

Comment (ravisuhag) [inline pkg/namespace/service.go:141]: @rahmatrhd method name could be just `encrypt` and `decrypt`.

**Comment by whoAbhishekSah** [comment]
The POST namespace API is not returning the id in the response payload.

**Comment by whoAbhishekSah** [comment]
There is test coverage missing on encrypt/decrypt function in the namespace service. Let's not miss coverage on core business logic. 
Screenshot 2022-03-15 at 12 17 07 PM

**Pull request merged**